netbout-web/src/test/java/com/netbout/rest/TkAppTest.java

```diff
@@ -155,6 +155,34 @@ public void redirectsToLoginAndSetsCookie() throws Exception {
     }
 
     /**
+     * TkApp includes authenticated user alias in header.
+     * @throws Exception If there is some problem inside
+     * @checkstyle MultipleStringLiteralsCheck (20 lines)
```

Comment:
@kujtimiihoxha I suggest to just use another `loc` and `auth`, or extract them to constants, and get rid of this checkstyle suppression.